apps/core/scenes/bookingDetail/sectorDetails/segments/SegmentMap.js

```diff
@@ -62,15 +129,15 @@ class SegmentMap extends React.Component<Props> {
         }}
         zoomEnabled={false}
         rotateEnabled={false}
+        pitchEnabled={false}
         onMapReady={this.fit}
       >
-        {coordinates.map(coordinate => (
-          <MapView.Marker
-            key={stringify(coordinate)}
-            coordinate={{
-              latitude: coordinate?.lat,
-              longitude: coordinate?.lng,
-            }}
+        {locations.map(location => (
+          <SegmentMapMarker
```

Comment:
Would it make sense to only pass `location` prop? and then deal with destructuring in the component?

Reply:
I prefer it this way, since if we are to rename any property in the location object I don't need to change anything in `SegmentMapMarker` component.